src/modules/GroupChannel/components/GroupChannelUI/index.tsx

```diff
@@ -40,6 +41,10 @@ export const GroupChannelUI = (props: GroupChannelUIProps) => {
       requestedChannelUrl={channelUrl}
       loading={loading}
       isInvalid={channelUrl && !currentChannel}
+      renderMessageInput={() => (
+        props.renderMessageInput?.()
+        ?? <MessageInputWrapper {...props} />
+      )}
```

Comment:
<img width="591" alt="image" src="https://github.com/sendbird/sendbird-uikit-react/assets/26326015/7f541337-35cd-4612-a3af-0be164bbd667">

The changes in this PR may not be directly related, but `GroupChannelUIView` does contain the basic components of `GroupChannel`, despite being in the presentation layer.

While it may seem convenient to use, it might be confusing for someone working on this code without prior knowledge of the collection operations.

@HoonBaek What are your thoughts on using it explicitly, as done in the code changes by Liam?

Reply:
Hoon will handle this as a separate PR.